cedar-policy-validator/src/lib.rs

```diff
@@ -127,6 +134,33 @@ impl Validator {
         )
     }
 
+    #[cfg(feature = "level-validate")]
```

Comment:
Super nit-pick: Can we move relevant code to a module (file)?

Reply:
agree. That might be nice

Reply:
Done.